templates/articles/to-translate.html

```diff
@@ -3,41 +3,47 @@
 {% block body_class %}translate-list-page{% endblock %}
 {% block body %}
 <section>
-  <h1>Articles to translate in {{ current_language[1] }}</h1>
 
-  <p>Change language:</p>
-  <ul>
-    {% for language in get_languages() %}
-      {% if current_language[0] != language[0] %}
-        <li><a href={{ url_for('articles.to_translate') }}?in={{language[0]}}>{{ language[1] }}</a></li>
-      {% endif %}
-    {% endfor %}
-  </ul>
+  <form method=get action=.>
+    <h1>
+      Articles to translate from
+      <select id=from name=from required>
+        {% for language in get_languages() %}
```

Comment:
Could we do `{% for language_code, language_name in get_languages()  %}` or something like it for readability?